### PR TITLE
fixing unstable tests -- upgraded version of rdbms

### PR DESCRIPTION
**Pull request by zikalino**
##### SUMMARY
Upgrading version of rdbms
Reenabling PostgreSQL and MySQL tests

##### ISSUE TYPE
 - Bugfix Pull Request

##### COMPONENT NAME
azure_rm_mysqlserver
azure_rm_mysqldatabase
azure_rm_postgresqlserver
azure_rm_postgresqldatabase

##### ANSIBLE VERSION
2.5

### CODE REVIEW

**Comment by zikalino** [comment]
@mattclay while working on this pr i have noticed that version of *msrest* package installed when using ansible normally was not the same as in the test environment.
according to requirements file, the latest version should be installed, but for some reason in integration test environment version was older (i am not sure what version exactly), so i had to force latest version. do you have any idea why it could happen?

File: test/integration/targets/azure_rm_mysqldatabase/tasks/main.yml [hunk 1]
```diff
@@ -8,8 +8,8 @@
     resource_group: "{{ resource_group }}"
     name: mysqlsrv{{ rpfx }}
     sku:
-      name: MYSQLB50
-      tier: basic
+      name: GP_Gen4_2
```
Comment (yungezz) [inline test/integration/targets/azure_rm_mysqldatabase/tasks/main.yml:11]: what's the difference between those sku?

**Review by yungezz — APPROVED** [review]

**Comment by yungezz** [comment]
shipit

**Comment by yuwzho** [comment]
shipit

**Comment by yungezz** [comment]
@zikalino   does this need backport to stable-2.6?

**Comment by mattclay** [comment]
@zikalino CI uses a docker container which already has the Azure requirements pre-installed. Since the requirement was `msrest!=0.4.15`, the version installed was whatever was current when the container was last built. If there is a minimum version requirement, this must be expressed in the requirements file. It is not safe to assume the latest version will be installed.

**Comment by zikalino** [comment]
@mattclay how frequently is that Docker container updated?
Right not I have noticed that newest version of *azure-common*  started causing problems in some modules, and seems like CI is not failing yet.

btw, could you approve this PR so we can merge?

**Comment by mattclay** [comment]
@zikalino The container isn't regularly updated. Having the requirements in the container up-to-date is simply an optimization for CI. Tests should not rely on particular requirements or versions being pre-installed.

**Pull request closed without merging**